ReactApp/src/index.css

```diff
@@ -93,36 +104,42 @@ h4 {
 
 .block-editor-inserter .block-editor-inserter__toggle {
 	background: black;
-	color: white;
-	/* FIXME: cleanup this mess */
+	color: white !important;
 	clip-path: inset(4px 4px 4px 4px);
 	margin: 4px 12px;
 	margin-right: 8px;
 	border-radius: 24px !important;
 }
 
+.block-editor-inserter__menu .block-editor-tabbed-sidebar__close-button {
+	display: none;
+}
+
 .block-editor-block-contextual-toolbar {
 	border-bottom: 0 !important;
 	/* TODO: fixme */
 }
 
 .gbkit-editor-toolbar_settings_icon {
-	background-repeat: no-repeat;
-	background-position: center;
-	background-image: url('data:image/svg+xml;utf8,<?xml version="1.0" ?><svg enable-background="new 0 0 32 32" id="Glyph" version="1.1" viewBox="0 0 32 32" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M27.526,18.036L27,17.732c-0.626-0.361-1-1.009-1-1.732s0.374-1.371,1-1.732l0.526-0.304  c1.436-0.83,1.927-2.662,1.098-4.098l-1-1.732c-0.827-1.433-2.666-1.925-4.098-1.098L23,7.339c-0.626,0.362-1.375,0.362-2,0  c-0.626-0.362-1-1.009-1-1.732V5c0-1.654-1.346-3-3-3h-2c-1.654,0-3,1.346-3,3v0.608c0,0.723-0.374,1.37-1,1.732  c-0.626,0.361-1.374,0.362-2,0L8.474,7.036C7.042,6.209,5.203,6.701,4.375,8.134l-1,1.732c-0.829,1.436-0.338,3.269,1.098,4.098  L5,14.268C5.626,14.629,6,15.277,6,16s-0.374,1.371-1,1.732l-0.526,0.304c-1.436,0.829-1.927,2.662-1.098,4.098l1,1.732  c0.828,1.433,2.667,1.925,4.098,1.098L9,24.661c0.626-0.363,1.374-0.361,2,0c0.626,0.362,1,1.009,1,1.732V27c0,1.654,1.346,3,3,3h2  c1.654,0,3-1.346,3-3v-0.608c0-0.723,0.374-1.37,1-1.732c0.625-0.361,1.374-0.362,2,0l0.526,0.304  c1.432,0.826,3.271,0.334,4.098-1.098l1-1.732C29.453,20.698,28.962,18.865,27.526,18.036z M16,21c-2.757,0-5-2.243-5-5s2.243-5,5-5  s5,2.243,5,5S18.757,21,16,21z" id="XMLID_273_"/></svg>');
-	margin-left: 4px;
+	font-size: 34px !important;
```

Comment:
There are a few[ "magic numbers"](https://en.wikipedia.org/wiki/Magic_number_(programming)) in the styles currently. In a separate PR later, I posit we should make a pointed effort to explain them through variable names, design systems, code comments, etc.

Reply:
I agree! To be honest this whole file needs to be updated and possibly split. I think we can work on that separately addressing other issues like the `!important` usage mentioned in this [comment](https://github.com/wordpress-mobile/GutenbergKit/pull/21#discussion_r1744350860).